ontology/yaml/resources/LIGHTING/entity_types/GENERALTYPES.yaml

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+LT:
```

Comment:
Why not just LIGHT? (For reference, the acronyms used in HVAC namespace are very commonly used; are these acronyms conventional within the lighting space?)

Reply:
I'm trying to reuse the abbreviation LT from the Google naming convention so that there is a direct mapping. Light is not a technical term, LUMINAIRE or LIGHT_FITTING could be an alternative if we want to use a more verbose term.

Reply:
i think LT is fine, just wanted to make sure there wasnt a more general acronym already.